.circleci/config.yml

```diff
@@ -471,14 +477,35 @@ jobs:
       - store_artifacts:
           path: "test/integration/render-tests/index.html"
 
+  prepare-windows:
+    executor:
+      name: win/default
+    working_directory: ~/mapbox-gl-js
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn run build-dev
+      - run:
+          name: Clean up workspace to persist faster
+          command: |
+            Remove-Item .git -Recurse -Force;
+            Remove-Item node_modules/flow-bin -Recurse -Force;
+            Remove-Item node_modules/tap -Recurse -Force;
+            Remove-Item node_modules/gl -Recurse -Force;
+            Remove-Item node_modules/@mapbox/mvt-fixtures/real-world/osm-qa-* -Recurse -Force;
+            Remove-Item node_modules/react-devtools-core -Recurse -Force;
+            Remove-Item node_modules/@octokit -Recurse -Force;
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - mapbox-gl-js
 
   test-render-windows-chrome-dev:
     executor:
       name: win/default
-    parallelism: 5
+    parallelism: 4
```

Comment:
I'm curious as to how you decided on 4. Do we seem to be gaining a small performance boost that allows the same completion time with a smaller number of runs?

Reply:
Yes, I thought I'd reduce the parallelism to compensate for less work — the more parallel jobs, the more resources are used, even if it completes faster. Also, since I'm adding a prepare job, it results in the same amount of jobs spawned.